Sources/SmileID/Classes/DocumentVerification/Model/DocumentCaptureModel.swift

```diff
@@ -313,6 +345,13 @@ class DocumentCaptureViewModel: ObservableObject, JobSubmittable, ConfirmationDi
         }
     }
 
+    @objc func manualCapture() {
+        DispatchQueue.main.async {
+            self.rectangleAspectRatio = self.maxAspectRatio
```

Comment:
We should keep detecting the rectangle (and auto-capture) if we can, even if manual capture button is enabled

Reply:
We are still detecting rectangles at this point. This sets the value of the rectangle aspect ratio to the max so that the guide fills the entire camera view finder. If a rectangle is detected after this, the rectangleAspectRatio will be updated.